.gitignore

```
@@ -0,0 +1,6 @@
+/kotlin/target
+/java/target
+/scala/project
+/scala/target
+/javascript/target
+/javascript/node_modules
```

Add missing gitignore for typescript:
```suggestion
/javascript/node_modules
/typescript/target
/typescript/node_modules
```